WordPress/Classes/ViewRelated/Tools/Time Zone/Views/TimeZoneTableViewCell.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+class TimeZoneTableViewCell: WPTableViewCell {
+
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        setupTimeZoneLabel(label)
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .body)
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
```

Comment:
As we're using the descriptor and size directly, we can just do `label.font = .preferredFont(forTextStyle: .body)` etc.